README.md

```diff
@@ -0,0 +1,217 @@
+o-layout [![Circle CI](https://circleci.com/gh/Financial-Times/o-layout/tree/master.svg?style=svg)](https://circleci.com/gh/Financial-Times/o-layout/tree/master)
+=================
+
+This component provides a responsive full page layout for internal documentation and tooling.
+It provides:
+- consistent design (typography, colors),
+- ease of set up (header, footer, sidebar, navigation)
+
+- [Usage](#usage)
+	- [Markup](#markup)
+		- [Layout Base](#layout-base)
+		- [Navigation and Content](#navigation-and-content)
+		- [Asides](#asides)
+		- [Tables](#tables)
+	- [Sass](#sass)
+- [Contact](#contact)
+- [Licence](#licence)
+
+
+## Usage
+`o-layout` is pure HTML and CSS, and relies on all of the sections within the markup to lay the page out correctly.
+
+In essence, the `o-layout` provides a grid that looks like this:
+```
+┌————————————————————————————┐
+| 		  HEADER 	  	|
+├————————————————————————————┤
+|		|		  		 |
+|  SIDE  |	 MAIN	 	 |
+|  BAR   |	 CONTENT	   |
+|		|			  	 |
+├————————————————————————————┤
+| 	  	FOOTER		   |
+└————————————————————————————┘
```

Comment:
Spacing on this is borked. I think it's probably due to a mix of tabs and spaces. Nearly always safest to use spaces along in ASCII art :)